Tests/Runner/StateFileTest.php

```diff
@@ -159,6 +160,29 @@ public function testUpdateStateChange()
             new NullFilter()
         );
         $stateFile->storeState(["state" => "fooBar", "#foo" => "bar"]);
+        $stateFile->persistState();
+    }
+
+    public function testUpdateStateChangeNoPersist()
```

Comment:
super název testu, co přejmenovat `StateFile::storeState` na `StateFile::updateState`? Mmch, `*State` v nátzvech metod by se mohlo vyhodit. `StateFile::update`, `StateFile::persist`, ten `*File` by se taky vlastně mohl vyhodit :D

Reply:
prejmenoval jsem to na `stashState` z toho je snad jasny, ze to je jen nejaky docasny zachyceni a potrebuje to jeste nejaky persist

Reply:
test metody jsem prejmenoval trosku